lib/OpenQA/WebAPI.pm

```diff
@@ -242,6 +242,8 @@ sub startup {
     # admins accessible tables
     $admin_r->get('/users')->name('admin_users')->to('user#index');
     $admin_r->post('/users/:userid')->name('admin_user')->to('user#update');
+    $admin_r->post('/users')->name('admin_user_create')->to('user#create');
+    $admin_r->delete('/users/:userid')->name('admin_user_delete')->to('user#delete');
```

Comment:
Like I've mentioned in the chat: These routes should better be proper API routes so they are conveniently usable via the clients as well (and not just limited to the scope of the web UI).